Anlab.Mvc/Client/components/TestList.tsx

```diff
@@ -68,9 +68,12 @@ export class TestList extends React.Component<ITestListProps, ITestListState> {
                         <td>
                             <Checkbox checked={selected} onChange={e => this.onSelection(item, e)} />
                         </td>
-                        <td><div className="analysisTooltip" data-toggle="tooltip" title={item.notes}>{item.analysis}</div></td>
+                        <td>{item.analysis}</td>
                         <td>{item.id}</td>
                         <td><NumberFormat value={priceDisplay} displayType={'text'} thousandSeparator={true} decimalPrecision={true} prefix={'$'} /></td>
+                        <td width="5%">
+                            {item.notes ? <i className="fa fa-info-circle" aria-hidden="true" id="analysisTooltip" data-toggle="tooltip" title={item.notes}></i> : ""}
```

Comment:
@laholstege there shouldn't ever be two elements in the DOM with the same ID, so you'll want to go with className here like it was before